cdap-ui/cypress/integration/pipeline.plugin.missing.spec.ts

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import{dataCy, loginIfRequired, getArtifactsPoll} from '../helpers';
+
+let headers = {};
+
+
+describe('Creating a pipeline', () => {
```

Comment:
Nit: Maybe this can be a little more specific, i.e. "creating a pipeline with a missing artifact" ?

Reply:
done!